_explore/queries/dependency-Info.gql

```diff
@@ -0,0 +1,17 @@
+query ($ownName: String!, $repoName: String!) {
+  repository(owner: $ownName, name: $repoName) {
+    languages(first: 1, orderBy: {field: SIZE, direction: DESC}) {
```

Comment:
```suggestion
  repository(owner: $ownName, name: $repoName) {
    nameWithOwner
    name
    languages(first: 1, orderBy: {field: SIZE, direction: DESC}) {
```
The name data may seem redundant, but it makes it easy to access these fields, and it is more in line with the base `repo-Info.gql` data. I think it's worth carrying over from the previous version of the dependency query. (But no need to re-run the script to add them to the data right away.)